lib/custom_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class CustomHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
+  def image(*args)
+    %(<a href="#{args.first}" target="_blank">#{super}</a>)
```

Comment:
I personally don't like stuff getting opened in new tabs.. especially on mobile. Maybe this is normal though? I dunno.